src/main/java/frc/robot/commands/GoHomeSequence.java

```diff
@@ -20,12 +20,12 @@ public GoHomeSequence(Climb climb, Elevator elevator, Arm arm, Claw claw, Lights
         /* do everything that can be done instantly first. we will have to check with design about interferance zones, but doing it in this order should be optimal and not require zone checking */
         new InstantCommand(() -> lights.setLEDColor(LightCode.HOME)),
         new InstantCommand(() -> claw.stopMotors()),
+        new InstantCommand(() -> elevator.setDesiredPosition(ElevatorHeight.HOME)),
+        new WaitUntilCommand(() -> elevator.atElevatorPosition(ElevatorHeight.ARM_CLEAR_OF_CLIMB)),
         new InstantCommand(() -> arm.setDesiredPosition(ArmPosition.HOME)),
-        new WaitUntilCommand(arm::atDesiredArmPosition),
+        new WaitUntilCommand(() -> {return elevator.atDesiredPosition() && arm.atDesiredArmPosition();}),
```

Comment:
Okay so I realized that this could be made slightly more efficient if we add back the ArmPosition.ARM_CLEAR_OF_CLIMB and only wait until the arm is there before we start moving the climb but idk if that is "worth it"

also, i could just be slightly mistaken about the interference zones in general